File: PerfectNumber/PerfectNumber.js
// Find given number is perfect number or not

// n = parseInt(prompt("Enter number: "))
n = 282
sum = 0
for (let i = 1; i <= n / 2; i++) {
    if (n % i == 0) {
        sum += i
    }
}
if (sum == n) {
    console.log(`${n} is a Perfect number`)
}
else {
    console.log(`${n} is not a Perfect number`)
}

File: PerfectNumber/PerfectNumber.c
// WAP to check given number is Perfect or not

#include <stdio.h>
int main()
{
    int n, i, sum = 0;
    printf("Enter number: ");
    scanf("%d", &n);
    for (i = 1; i <= n / 2; i++)
    {
        if (n % i == 0)
        {
            sum += i;
        }
    }
    if (sum == n)
    {
        printf("%d is a Perfect Number", n);
    }
    else
    {
        printf("%d is not a Perfect Number", n);
    }
    return 0;
}